### DISCUSSION TITLE
What should I do to my github copilot bill after I received my student developer pack?

### DISCUSSION THREAD

**Discussion by somebody-someone** [community/community#42751] [GitHub Education] [answered]
I started github copilot before receiving my student developer pack, and now my application has been approved. So what should I do now? Should I cancel my original bill? Or it will continue to be paid monthly? If I cancel my bill for the copilot, will I be able to use the copilot?

**Comment by DSDmark** [comment, marked as answer]
Hi @somebody-someone,

If you have been approved for the GitHub Student Developer Pack, you should be able to use GitHub Copilot for free as long as you are a student and your pack is still active.

**Reply by somebody-someone** [reply]
Thanks for your answer!

**Reply by chapi1234** [reply]
I am a verified student having problems with gettting copilot pro. it says you using free plan and you reached your monthly limit with 50 message

**Reply by jechave** [reply]
Hi, I have the same problem. Could you solve it? Thanks.

**Reply by chapi1234** [reply]
yeah redem it if you didn't. after the redem it will work 👍

**Reply by lucaspsrrio** [reply]
How to redeem? Thanks for answering!

**Reply by chapi1234** [reply]
there is a reedem link but curently I don't have it. just watch a youtube video it is so simple 👍

**Comment by queenofcorgis** [comment]
Hi @somebody-someone ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your additional post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have removed the duplicate post, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

**Reply by anksbhat** [reply]
I applied for student GitHub education pack twice. It was accepted first and then rejected the second time I applied accidentally. What do I do  now? I am not able to access free features

**Comment by jechave** [comment]
ok, tks. I sorted it, wasn't obvious where to do it.
Cheers.

**Comment by jechave** [comment]
I'm not sure of the exact steps, it was kind of random... I think I went to
"Your Copilot", there it said that I had copilot free active, and I could
click there to get the upgrade, and then somehow I got the upgrade with a
bill of 0 because of the educational package... I'm sorry if I can't be
more precise, I don't remember!
